cmd/osbuild-tests/main_test.go

```diff
@@ -21,24 +21,72 @@ import (
 	"testing"
 )
 
+func TestComposeCommands(t *testing.T) {
+	// common setup
+	tmpdir := NewTemporaryWorkDir(t, "osbuild-tests-")
+	defer tmpdir.Close(t)
+
+	bp := blueprint.Blueprint{
+		Name:        "empty",
+		Description: "Test empty blueprint in toml format",
+		Packages: []Package{{Name: "bash", Version: "*"}},
```

Comment:
I think this needs to be `Packages: []blueprint.Package{{Name: "bash", Version: "*"}},`